src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/DeliverySessionService.kt

```diff
@@ -221,7 +221,9 @@ class DeliverySessionService(
       deliusAppointmentId = deliusAppointmentId,
       createdAt = OffsetDateTime.now(),
       createdBy = existingAppointment?.createdBy ?: authUserRepository.save(updatedBy),
-      referral = existingAppointment?.referral ?: session.referral
+      referral = existingAppointment?.referral ?: session.referral,
+      superseded = session.appointments.size> 0
```

Comment:
What if there are two separate appointments(both will have superseded=false), then this case would be true.